Source/WebCore/dom/EventTargetFactory.in

```diff
@@ -2,6 +2,7 @@ namespace="EventTarget"
 factoryFunction=toJS
 useNamespaceAsSuffix=true
 
+
```

Comment:
This is necessary to force a rebuild of `EventTargetInterfaces.h`.

Reply:
Hmm, it seems like dependencies are not setup correctly then in the build system somewhere? Might be good to file a follow-up for this.